README.md

```diff
@@ -126,6 +126,32 @@ createP.then(function(response) {
 `response` is a JSON object containing the FaunaDB response. See the JSDocs for
 `faunadb.Client`.
 
+#### ES6 modules
+
+This is recommended way to inject faunadb-js. Many build tools supports three shaking which relies on the static structure of ES6 module syntax, i.e. `import` and `export`.
+
+```
+import { Client, query } from 'faunadb'
+
+const client =new Client({secret: 'your_key_here'})
+client.query(query.Now()).then(console.info)
+```
+
+Another way to import queries functions
+
+```
+import { Now, Select, Var, FaunaObject, FaunaFunction } from 'faunadb/query'
+query.Object has been replaced to FaunaObject
+query.Function has been replaced to FaunaFunction
+```
+
```

Comment:
Here is a great place to remind users that `Function` and `Object` are reserved words in JavaScript, so if they want to import as non-namespaced names, they need to be imported with custom names, such as `FFunction` and `FObject`.